### PR TITLE
Solved SignOut Page Error

### PR DESCRIPTION
**Pull request by Subhraneel77**
Fixes #
When we will Sign Out then we will be redirected to Sign In Page.

#### Describe the changes you have made in this PR -

Just added the url of the Sign in page inside the header.html.

### CODE REVIEW

File: app/assets/javascripts/webpush.js.erb [hunk 1]
```diff
@@ -28,10 +28,17 @@ function testWebPush() {
 }
 
 function checkPermissions() {
-  if (Notification.permission === "granted") {
+  if (Notification.permission !== 'denied'")
```
[the remark this answers is not in the document]
Comment (Subhraneel77) [reply app/assets/javascripts/webpush.js.erb:31]: Okay Looking Forward to it

**Comment by vedant-jain03** [comment]
Accidentally close the the PR's sorry for that,
@Subhraneel77 Can you please provide meaningful title to PR

**Comment by Subhraneel77** [comment]
Sigh Out Issue Has been resolved

**Comment by tachyons** [comment]
The files changed seems to be un related, please reopen the PR once the issue is addressed

**Pull request closed without merging**